shell/platform/android/BUILD.gn

```diff
@@ -116,6 +116,9 @@ embedding_jar_path = "$root_out_dir/$embedding_jar_filename"
 embedding_sources_jar_filename = "$embedding_artifact_id-sources.jar"
 embedding_source_jar_path = "$root_out_dir/$embedding_sources_jar_filename"
 
+# Version string used by genrate_pom_file.py, as well
+_maven_version_string = "1.0.0-$engine_version"
```

Comment:
what about it just uses 1.0.0 since for local engine, there's only one version. Also, it should help simplify the logic in the tool.

Reply:
I'm not clear on what that would let me simplify?  I'd still want a variable for it because it's used in multiple places right?

Reply:
If this is for local engine only, I think some fixed version should work.

Reply:
Right but either way we have to pass it into the script that generates the files with the version string.  Am I just missing what this would simplify?

Reply:
the tool would need to know about the engine version, so we will need a new flag that only works on android.

Reply:
Doesn't the tool just read that from the `.pom` files?  We'll still have the one at the root of the repo

Reply:
The `.pom` file is in a directory that also includes the version in the filename. e.g. http://download.flutter.io/io/flutter/arm64_v8a_debug/1.0.0-5f3b749e92d348a04a4073e1adadd3f8eb114650/arm64_v8a_debug-1.0.0-5f3b749e92d348a04a4073e1adadd3f8eb114650.pom

Reply:
But we still have it at the root of the out directory as well, which is where the tool currently gets the version from.

If we simplify this here, we should really just create a separate pom file that just has the simplified version - but that seems to complicate the build process a bit more for the sake of removing existing code from the tool.

Reply:
I see. That works. This way, we still parse the root `.pom` file, but then Gradle uses the one from the repo.